src/components/TopBar/OhmMenu.jsx

```diff
@@ -0,0 +1,106 @@
+import { useState } from "react";
+import { addresses } from "../../constants";
+import { getOhmTokenImage, getSohmTokenImage } from "../../helpers";
+import { useSelector } from "react-redux";
+import { Link, SvgIcon, Popper, Button, Paper, Typography, Divider, Box } from "@material-ui/core";
+import { ReactComponent as InfoIcon } from "../../assets/icons/v1.2/info-fill.svg";
+import { ReactComponent as ArrowUpIcon } from "../../assets/icons/v1.2/arrow-up.svg";
+import "./ohmmenu.scss";
+
+const TOKEN_DECIMALS = 9;
```

Comment:
I personally prefer keeping global variables in a single file (e.g. constants) and then importing.

Reply:
facts. this is likely from copypasta of the original code. making a note to include this in next update